content/Products/OpenshiftMonitoring/alerting.md

```diff
@@ -0,0 +1,30 @@
+# Alerting guidelines
```

Comment:
When do we have to follow this? in a regular basis? Will there be a pointer to this?

Reply:
I'd not be the best timing for the exercise, but we can add a reminder for it in https://github.com/openshift/cluster-monitoring-operator/blob/master/.github/ISSUE_TEMPLATE/release.md